utils/build/declarations.d.ts

```diff
@@ -24,3 +49,148 @@ declare module '@zendeskgarden/react-grid';
+  export const zdColorSecondaryAzureM400: string;
+  export const zdColorSecondaryAzureM600: string;
+  export const zdColorSecondaryCrimsonM400: string;
+  export const zdColorSecondaryCrimsonM600: string;
+  export const zdColorSecondaryFuschiaM400: string;
+  export const zdColorSecondaryFuschiaM600: string;
+  export const zdColorSecondaryLemonM400: string;
+  export const zdColorSecondaryLemonM600: string;
+  export const zdColorSecondaryLimeM400: string;
+  export const zdColorSecondaryLimeM600: string;
+  export const zdColorSecondaryMintM400: string;
+  export const zdColorSecondaryMintM600: string;
+  export const zdColorSecondaryOrangeM400: string;
+  export const zdColorSecondaryOrangeM600: string;
+  export const zdColorSecondaryPinkM400: string;
+  export const zdColorSecondaryPinkM600: string;
+  export const zdColorSecondaryPurpleM400: string;
+  export const zdColorSecondaryPurpleM600: string;
+  export const zdColorSecondaryRoyalM400: string;
+  export const zdColorSecondaryRoyalM600: string;
+  export const zdColorSecondaryTealM400: string;
+  export const zdColorSecondaryTealM600: string;
+  export const zdColorChatOrange: string;
+  export const zdColorConnectRed: string;
+  export const zdColorExploreBlue: string;
+  export const zdColorGuidePink: string;
+  export const zdColorMessageGreen: string;
+  export const zdColorSellGold: string;
+  export const zdColorSupportGreen: string;
+  export const zdColorTalkYellow: string;
+  export const zdFontFamilyMonospace: string;
+  export const zdFontFamilySystem: string;
+  export const zdFontSizeSm: string;
+  export const zdFontSizeMd: string;
+  export const zdFontSizeLg: string;
+  export const zdFontSizeXl: string;
+  export const zdFontSizeXxl: string;
+  export const zdFontSizeXxxl: string;
+  export const zdFontSizeSmMonospace: string;
+  export const zdFontSizeMdMonospace: string;
+  export const zdFontSizeLgMonospace: string;
+  export const zdFontWeightThin: string;
+  export const zdFontWeightExtralight: string;
+  export const zdFontWeightLight: string;
+  export const zdFontWeightRegular: string;
+  export const zdFontWeightMedium: string;
+  export const zdFontWeightSemibold: string;
+  export const zdFontWeightBold: string;
+  export const zdFontWeightExtrabold: string;
+  export const zdFontWeightBlack: string;
+  export const zdFontWeightUltralight: string;
+  export const zdFontWeightUltrabold: string;
+  export const zdFontWeightHeavy: string;
+  export const zdLineHeightSm: string;
+  export const zdLineHeightMd: string;
+  export const zdLineHeightLg: string;
+  export const zdLineHeightXl: string;
+  export const zdLineHeightXxl: string;
+  export const zdLineHeightXxxl: string;
+  export const zdSpacingXxs: string;
+  export const zdSpacingXs: string;
+  export const zdSpacingSm: string;
+  export const zdSpacing: string;
+  export const zdSpacingLg: string;
+  export const zdSpacingXl: string;
+  export const zdSpacingXxl: string;
+}
```

Comment:
😢

Reply:
This isn't needed for the code to compile, but with creating some of these CSS-in-JS areas it helped a lot. Once we include these values in `react-theming` it will no longer be necessary.